src/graphql/operations/vote.ts

```diff
@@ -3,7 +3,7 @@ import { formatVote } from '../helpers';
 
 export default async function(parent, { id }) {
   const query = `
-    SELECT v.*, spaces.settings FROM votes v
+    SELECT v.*, spaces.settings, spaces.created_at, spaces.updated_at FROM votes v
```

Comment:
The column aliases (`space_created_at`, `space_updated_at`) are missing here but are expected in `formatVote()`.

Reply:
Uhm I'm a bit unsure about what's happening here, I see them present in my repo branch...  

https://github.com/stormsson/snapshot-hub/blob/%23166-created-updated-in-graphql/src/graphql/operations/vote.ts